Cargo.toml

```
@@ -38,6 +38,8 @@ color-backtrace = { version = "0.4", default-features = false, optional = true }
 counter = "0.4"
 atty = { version = "0.2", default-features = false, optional = true } 
 git-testament = { version = "0.1", optional = true }
+rustyline = "6.1.2"
+rustyline-derive = "0.3.1"
```

I would prefer for this to be an optional feature activated by default. That allows projects wanting to use rcc programmatically not to have to compile `rustyline`.
```suggestion
rustyline = { version = "6.1.2", optional = true }
rustyline-derive = { version = "0.3.1", optional = true }
```

You'll also have to add a `repl` feature under `[features]`, then add that features to `default = [ ... ]`.